apps/condo/domains/billing/schema/AllResidentBillingReceiptsService.js

```diff
@@ -172,10 +205,18 @@ const AllResidentBillingReceiptsService = new GQLCustomSchema('AllResidentBillin
                             get(receipt, ['context', 'organization', 'id']) === organization ),
                         currencyCode: get(receipt, ['context', 'integration', 'currencyCode'], null),
                         file,
-                        isPayable: receipt.isPayable,
+                        isPayable,
                     })
                 })
 
+                // cache already paid payments
+                const paymentsCache = await  find('Payment', {
+                    organization: { id_in: [... new Set(processedReceipts.map(item => get(item, ['serviceConsumer', 'organization'])))] },
+                    accountNumber_in: [... new Set(processedReceipts.map(item => get(item, ['serviceConsumer', 'accountNumber'])))],
+                    period: minPeriod,
```

Comment:
why minPeriod?
minPeriod is a minimum period for consumer's receipts
there can be 2 different receipts: for overhaul and housing with different periods